handlerUser.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+)
+
+func handlerUser(w http.ResponseWriter, r *http.Request) {
```

Comment:
Handler name could be more on point to what it does? It's not handling user, is it?

Reply:
Oh ya. Will fix.